metalsapi/package.json

```diff
@@ -1,15 +1,46 @@
 {
   "name": "@chainlink/metalsapi-adapter",
   "version": "0.0.1",
```

Comment:
Should we be bumping versions for all of these TS migrations?

Reply:
Yeah, I think we should!

Reply:
Since all versions are now set to `0.0.1`, let's wait until everything is finished/merged, and batch-update all TS migrated ones, or all TS ones in general.